src/impls.rs

```diff
@@ -167,4 +167,40 @@ pub fn next_u64_via_fill<R: Rng+?Sized>(rng: &mut R) -> u64 {
     impl_uint_from_fill!(rng, u64, 8)
 }
 
+/// Implement `fill_bytes` via `try_fill` with implicit error handling.
+pub fn fill_via_try_fill<R: Rng+?Sized>(rng: &mut R, dest: &mut [u8]) {
```

Comment:
I still feel a bit uneasy about this function. We had to pick some defaults for how many times to retry or how long to wait in case of an error, that may not suite every RNG.

What do you think about moving it to `os.rs` and not making it public?

Reply:
At the moment it's not public. I see your point though, we could simply force every use-case to have its own custom implementation instead of having this one. I'm not sure.